packages/ui-components/jest-runner-eslint.config.js

```diff
@@ -1,5 +0,0 @@
-module.exports = {
-  cliOptions: {
-    format: "codeframe",
```

Comment:
>The codeframe formatter is no longer part of core ESLint. Install it manually with `npm install -D eslint-formatter-codeframe`

We could install the formatter separately, but the package isn't maintained. https://www.npmjs.com/package/eslint-formatter-codeframe